client/src/types/data.ts

```diff
@@ -117,6 +126,9 @@ export type CO2T = Static<typeof CO2RT>;
 /** Value type of accelerometer sensor data */
 export type AccelerometerT = Static<typeof AccelerometerRT>;
 
+/** Value type of aneomometer sensor data */
+export type AnemometerRT = Static<typeof AnemometerRT>;
```

Comment:
```suggestion
export type AnemometerT = Static<typeof AnemometerRT>;
```
we already define a variable called `AnemometerRT` above, here we define a type and although we could still use the same name, it won't make sense since it's not Anemometer**RT**. The 'RT' stands for RunType and is not strictly a `type` in TypeScript. Some [background reading](https://github.com/pelotom/runtypes) if you're interested to know what RunType is and why we use it. 

In short, some places in our code accept a **variable** that represents our data, so we use `AnemometerRT`, but in other places we need to use a **type** so we define a type called `AnemometerT`.